### PR TITLE
remove django-reversion dependency

### PR DESCRIPTION
**Pull request by rosskarchner**
I haven't been able to find any remaining code that uses [django-reversion](https://django-reversion.readthedocs.io/en/stable/), and the last Revision in the database has a 2014 timestamp.

~Potentially controversial: I included a data migration that removes Reversion's entries form the contenttypes table (and thus, permissions). It seems like Django's automatic detection of stale content types does *not* extend to apps that have been removed.~

## Removals

- references to django-reversion

## Notes

-

## Todos

-

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] Any *change* in functionality is tested
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

File: cfgov/core/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.contrib.contenttypes.models import ContentType
+
+
+def remove_reversion_ct(apps,schema_editor):
+    ContentType.objects.filter(app_label='reversion').delete()
```
Comment (chosak) [inline cfgov/core/migrations/0001_initial.py]: Can you refer to a model via import like this, in a migration? I think you might need to do something like

```py
ContentType = apps.get_model('contenttypes', 'ContentType')
```

to void errors.

Comment (rosskarchner) [reply cfgov/core/migrations/0001_initial.py]: hmm, it worked for me-- but this is probably cleaner.

**Comment by rosskarchner** [comment]
@chosak I've removed the migration

File: cfgov/core/migrations/0001_initial.py (hunk 1 above)
Comment (chosak) [reply cfgov/core/migrations/0001_initial.py]: Right, it'll work, but generally best practice in Django migrations is not to import models like that. The problem would be, if, say, in a future Django version the `ContentType` model has some different fields. This migration might run before the one modifying `ContentType`, which means that the state of the model via import from code would differ from the one in the DB. Using `apps.get_model` ensures that you always get the right model version at the time of the migration.

**Comment by rosskarchner** [comment]
I thought there was no such thing as an empty directory in git?

**Comment by rosskarchner** [comment]
Ah, nm, I see what you mean

**Comment by rosskarchner** [comment]
Deleting it seems to be harmless.

**Pull request merged**